databricks-iris/credentials.yml

```diff
@@ -0,0 +1,18 @@
+# Here you can define credentials for different data sets and environment.
+#
+# THIS FILE MUST BE PLACED IN `conf/local`. DO NOT PUSH THIS FILE TO GitHub.
+#
+# Example:
+#
+# dev_s3:
+#     client_kwargs:
+#         aws_access_key_id: token
+#         aws_secret_access_key: key
+#
+# prod_s3:
+#     aws_access_key_id: token
+#     aws_secret_access_key: key
+#
+# dev_sql:
+#     username: admin
+#     password: admin
```

Comment:
Should this be inside `conf/local`?

Reply:
Good spot, this file has been in the root of `pyspark-iris` since December 2020 and needs to be moved to `conf/local` in `pyspark-iris` also.

Reply:
In fact, I know why it is here. Placing it in `conf/local` means the `.gitignore` file in the root of `iris-databricks` (and `pyspark-databricks`) ignores it.

Since there is a note to the user to move it and there isn't an easy way to add it to VC otherwise, I think it should remain here.